templates/index.html

```diff
@@ -38,7 +38,7 @@ <h2><a href="{{ SITEURL }}/{{ article.url }}{% if not DISABLE_URL_HASH %}#{{ art
       {% if  article.featured_image %}
         <img src="{{ article.featured_image }}">
       {% endif %}
-      {{ article.summary }}
+      <p>{{ article.summary }}</p>
```

Comment:
I believe that multiple lines will create a <p> tag, so in this case, <p> inside another <p> is not that great, would a <div> work for the same purpose?

<p> tags could also get some css from somewhere, <div> is less likely to.

Reply:
It is as you state indeed.  The <div> works expected.

How to continue, with this patch, now?

Reply:
Not sure what you are using, but in git is basically to either create another commit, or amend the current one (in the same branch).

You can also close this PR and submit another one if it's easier.

Reply:
Or you can also use the option "Edit File" in this page.

Reply:
I editted the file, but the change is not showing up here...  There is probably another (new?) PR needed.